core/net6.0/proxy/Apache.OpenWhisk.Runtime.Common/Run.cs

```diff
@@ -49,12 +49,13 @@ public async Task HandleRequest(HttpContext httpContext)
 
             try
             {
-                string body = await new StreamReader(httpContext.Request.Body).ReadToEndAsync();
+                using StreamReader reader = new(httpContext.Request.Body);
```

Comment:
Let's be good citizens and dispose any managed resources created by the `StreamReader`.